cfgov/jinja2/v1/_includes/organisms/filterable-list.html

```diff
@@ -0,0 +1,195 @@
+                {% from 'molecules/info-unit-2.html' import info_unit with context %}
+                <div class="o-info-unit-group u-mb30" data-qa-hook="image-text-50-50">
+                    <div class="content-l">
+                        {% for post in posts %}
+                        <div class="content-l_col
+                                    content-l_col-1-2">
+                            {% set post_url = pageurl(post) %}
+                            {% if post.preview_image %}
+                                {% set photo = image(post.preview_image, 'original') %}
+                            {% endif %}
+                            {% set heading = '<a href="' ~ post_url ~ '"><h2 class="h3">' ~
+                                            post.preview_title ~
+                                            '</h2></a>'
+                            if post.preview_title else null %}
+                            {% set sub_heading = '<a href="' ~ post_url ~ '"><h2 class="h4">' ~
+                                                post.preview_subheading ~
+                                                '</h2></a>'
+                            if post.preview_subheading else null %}
+                            {% if post.secondary_link_url and post.secondary_link_text %}
+                                {% set links = [
+                                    {
+                                        'url': post.secondary_link_url,
+                                        'text': post.secondary_link_text
+                                    }
+                                ] %}
+                            {% endif %}
+                            {{ info_unit( {
+                                'image': {
+                                    'url': photo.url if photo else '/',
+                                    'alt': post.preview_image.alt if post.preview_image.alt else '',
+                                },
+                                'heading': heading,
+                                'sub_heading': sub_heading,
+                                'body': post.preview_description,
+                                'links': links or null,
+                                'link_image_and_heading': value.link_image_and_heading
+                            } ) }}
+                        </div>
+                        {% endfor %}
+                    </div>
+                </div>
+            {% else %}
+                {% import 'organisms/post-preview.html' as post_preview with context %}
+                {% for post in posts %}
+                    {{ post_preview.render(post, value) }}
+                {% endfor %}
+            {% endif %}
+
+            {# DISPLAY THE PAGINATOR, IF THERE ARE RESULTS OVER A COUNT. #}
+
+            {% set total_pages = posts.paginator.num_pages %}
+            {% set current_page = posts.number %}
+            {% if total_pages > 1 and current_page <= total_pages %}
```

Comment:
This condition seems superfluous?

```suggestion
            {% if total_pages > 1 %}
```

Reply:
Hmmmmm okay I've removed it in https://github.com/cfpb/cfgov-refresh/pull/4771/commits/9086a5fea3d934f9834f91151cadf56f384a4fb0

This is actually a direct duplicate from https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jinja2/v1/_includes/molecules/pagination.html#L26 that is necessary to hide the pagination container, which is kinda a bummer.

Reply:
It seems unnecessary for that to exist there. Seems like it should be up to the thing calling pagination to know if it actually should be calling it or not.

Reply:
Possibly, the method signature looks pretty clear there and it's nice it handles the arguments coming into it correctly.